### PR TITLE
Added a before submit handler using jQuery so that the widget works

### PR DESCRIPTION
**Pull request by reastyn**
Hey,
found that the plugin not working and with only one line of code it can work even now with newest intlTelInput.

Would you consider to add this quickfix to the live version so that the yii2 community can use this awesome widget?
Thanks

### CODE REVIEW

File: src/PhoneInput.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function init()
         PhoneInputAsset::register($this->view);
         $id = ArrayHelper::getValue($this->options, 'id');
         $jsOptions = $this->jsOptions ? Json::encode($this->jsOptions) : "";
-        $this->view->registerJs("$('#$id').intlTelInput($jsOptions);");
+        $this->view->registerJs("var input = $('#$id'); input.intlTelInput($jsOptions); input.parents('form').on('submit', function() { input.val(input.intlTelInput('getNumber')); });");
```
Comment (Borales) [inline src/PhoneInput.php]: @reastyn what if this widget is being used outside of form? (just a simple widget input)

Comment (Borales) [inline src/PhoneInput.php]: @reastyn I will check it, but I think every `registerJs` adds JS code to a single scope, so `var input` will override all previous phone widget, am I right?

Comment (reastyn) [inline src/PhoneInput.php]: Yeah you're right. My bad, would you consider pulling my newer commit?

File: src/PhoneInput.php [hunk 2]
```diff
@@ -26,7 +26,10 @@ public function init()
         PhoneInputAsset::register($this->view);
         $id = ArrayHelper::getValue($this->options, 'id');
         $jsOptions = $this->jsOptions ? Json::encode($this->jsOptions) : "";
-        $this->view->registerJs("var input = $('#$id'); input.intlTelInput($jsOptions); input.parents('form').on('submit', function() { input.val(input.intlTelInput('getNumber')); });");
+        $this->view->registerJs("$('#$id').intlTelInput($jsOptions);");
+        if($this->hasModel()) {
+            $this->view->registerJs("$('#$id').parents('form').on('submit', function() { $('#$id').val(input.intlTelInput('getNumber')); })");
```
Comment (ekussberg) [inline src/PhoneInput.php]: It doesn't work for me so i replaced the last "input" => $('#$id') and then it works like charm

**Pull request merged**